oauth2/hook.go

```diff
@@ -17,12 +17,30 @@ import (
 // AccessRequestHook is called when an access token is being refreshed.
 type AccessRequestHook func(ctx context.Context, requester fosite.AccessRequester) error
 
+// Requester is a token endpoint's request context.
+//
+// swagger:model requester
+type Requester struct {
```

Comment:
I created this type because `fosite.AccessRequester` has only functions and generated swagger was not very useful.

Reply:
👍 good call